### DISCUSSION TITLE
Is there any way to have SDL or introspection separately than the rest of the graphql (or graphql endpoint)

### DISCUSSION THREAD

**Discussion by shadikkhan** [Netflix/dgs-framework#1699] [Q&A]
Is there any way to have SDL or introspection separately than the rest of the graphql (or graphql endpoint)? I am trying to provide separate endpoint(may be) for introspection and rest normal call can go to graphql

**Comment by srinivasankavitha** [comment]
We don't currently have a way to separately configure the introspection endpoint. It all goes to /graphql or the configured graphql endpoint.
